docs/sources/flow/reference/components/otelcol.connector.servicegraph.md

```diff
@@ -0,0 +1,218 @@
+---
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.connector.servicegraph/
+labels:
+  stage: experimental
+title: otelcol.connector.servicegraph
+---
+
+# otelcol.connector.servicegraph
+
+{{< docs/shared lookup="flow/stability/experimental.md" source="agent" >}}
+
+`otelcol.connector.servicegraph` accepts span data from other `otelcol` components and 
+outputs metrics representing the relationship between various services in a system.
+A metric represents an edge in the service graph.
+Those metrics can then be used by a data visualization application (e.g. 
+[Grafana](https://grafana.com/docs/grafana/latest/explore/trace-integration/#service-graph))
+to draw the service graph.
+
+> **NOTE**: `otelcol.connector.servicegraph` is a wrapper over the upstream
+> OpenTelemetry Collector `servicegraph` connector. Bug reports or feature requests
+> will be redirected to the upstream repository, if necessary.
+
+Multiple `otelcol.connector.servicegraph` components can be specified by giving them
+different labels.
+
+This component is based on [Grafana Tempo's service graph processor](https://github.com/grafana/tempo/tree/main/modules/generator/processor/servicegraphs).
```

Comment:
`otelcol.connector.servicegraph` is a wrapper over Collector's `servicegraph` connector, which on the other hand is inspired by the Tempo service graph processor.

As far as I can tell, the Otel component doesn't import any Tempo code. 

I think the reason why they mention Tempo in the Otel wiki page is because (I think) the Otel component inherits the metric names which the Tempo processor in question uses. So any other UI will also have to use the same metric conventions.

Reply:
Maybe we can then remove the Tempo reference or amend this sentence to mention the inspiration of metric names.

Reply:
I would say... only include the reference to Tempo if it helps the reader understand the topic. If it's just a nod to Tempo, or extra info that isn't really going help with understanding... then we should remove it.